### PR TITLE
change: config.yaml always from the latest version of apisix

### PR DESCRIPTION
**Pull request by gxthrj**

### CODE REVIEW

File: example/README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 ### Run
 
 ```
+$ chmod +x config.sh
+$ sh config.sh
```
Comment (moonming) [inline example/README.md:5]: Does this change have anything to do with the title?

Comment (gxthrj) [reply example/README.md:5]: the `config.sh` is used to generate `config.yaml` from the latest version of APISIX

Comment (membphis) [reply example/README.md:5]: `config.sh` is a not good name. How about `fetch_config_yaml.sh` ?

Comment (gxthrj) [reply example/README.md:5]: ok, I create a issue here https://github.com/apache/incubator-apisix-docker/issues/67, will change the name later

**Pull request merged**